File: src/main/java/org/location/model/ItemForRent.java
package org.location.model;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class ItemForRent {
    private final String id;
    private final String name;
    private final String description;
    private final double pricePerDay;
    private Status status;
}

File: ItemForRent.js
const Status = require("./Status");

class ItemForRent {
  constructor(id, name, description, pricePerDay, status = Status.AVAILABLE) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.pricePerDay = pricePerDay;
    this.status = status;
  }
}

module.exports = ItemForRent;
